packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -62,6 +66,27 @@ def noop(b, opts):
 }
 
 
+def _apply_sysctl(setting, service):
+    try:
+        check_call(["sysctl", "-q", "-w", setting])
+    except CalledProcessError:
```

Comment:
Do we know what platforms we expect which ones to fail on? I'd rather not swallow typos here, and it doesn't seem that complicated to make it so we have a condition potentially around the ones we know don't work with particular kernel versions.